tests/spread/integration/coreutils-from-gnu/task.yaml

```diff
@@ -76,3 +76,29 @@ execute: |
   test "$(chroot "$rootfs_printf" printf "hello-%s\n" "world")" = "hello-world"
   test "$(chroot "$rootfs_printf" printf "number: %d\n" 42)" = "number: 42"
   test "$(chroot "$rootfs_printf" printf "float: %.2f\n" 3.14159)" = "float: 3.14"
+
+  # test cat
```

Comment:
non-blocking but this will be great to turn into variants when we figure out a general strategy.

Reply:
agreed. for now just went with the pattern that's there